contracts/factory/src/tests/config.rs

```diff
@@ -130,3 +135,81 @@ fn factory_successfully_inits_lp() {
         }
     );
 }
+
+#[test]
+#[should_panic(
+    expected = "Factory: Create Liquidity Pool: You are not authorized to create liquidity pools!"
+)]
+fn factory_fails_to_init_lp_when_authorized_address_not_present() {
+    let env = Env::default();
+    let admin = Address::from_string(&String::from_str(
+        &env,
+        "CBT4WEAHQ72AYRD7WZFNYE6HGZEIX25754NG37LBLXTTRMWKQNKIUR6O",
+    ));
```

Comment:
I used that `from_string` before I found out that they replaced `::random` with `::generate`. Use the latter.
```suggestion
    let admin = Address::generate(&env);
```